analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/ProgramState.cs

```diff
@@ -142,6 +142,8 @@ public ProgramState ResetOperations()
         public override string ToString() =>
             Equals(Empty) ? "Empty" + Environment.NewLine : SerializeExceptions() + SerializeSymbols() + SerializeOperations() + SerializeCaptures();
 
+        public static implicit operator ProgramState[](ProgramState from) => new[] { from };
```

Comment:
`new[] { from }` isn't trivial, it should go on the next line
```suggestion
        public static implicit operator ProgramState[](ProgramState from) => 
            new[] { from };
```

Reply:
And we should have UTs for that in `ProgramStateTest`